packages/react-core/src/components/Switch/__tests__/Switch.test.tsx

```diff
@@ -67,3 +68,11 @@ test('switch passes value and event to onChange handler', () => {
   input.simulate('change', { target: { checked: true } });
   expect(props.onChange.mock.calls[0][0]).toBe(true);
 });
+
+test('switch accepts React.Node', () => {
+const view = mount(<Switch id="switch-react-node" label={<span>on</span>} labelOff={<span>off</span>}></Switch>);
+  expect(view.find('span#switch-react-node-on').find('span').first().text()).toBe('on');
+  expect(view.find('span#switch-react-node-off').find('span').first().text()).toBe('off');
+  assertPropTypes(Switch.propTypes, { label: <span>on</span> }, 'prop', Switch.name);
```

Comment:
The reason this even works is because we are using a babel plugin to generate prop types. So `Switch.propTypes` does exist in `dist/esm/components/Switch/Switch.js`.

Reply:
ok, removed and added demo